packages/main/test/specs/Table.spec.js

```diff
@@ -371,6 +371,29 @@ describe("Table general interaction", () => {
 			assert.strictEqual(await selectionChangeCount.getProperty("value"), "5",  "Enter key over an already selected row should trigger selection-change event");
 		});
 
+		it("test selectAll functionallity in MultiSelect mode", async () => {
```

Comment:
could you also add a case, when the checkbox is checked, then you have unselected an item.